packages/big-design/src/components/Radio/styled.tsx

```diff
@@ -28,30 +29,41 @@ export const StyledRadio = styled.label<StyledRadioProps>`
   border-color: ${props => (props.checked ? props.theme.colors.primary40 : props.theme.colors.secondary30)};
   border-radius: ${({ theme }) => theme.borderRadius.circle};
   color: ${({ theme }) => theme.colors.white};
+  cursor: pointer;
   display: inline-block;
   height: ${({ theme }) => theme.spacing.large};
   position: relative;
-  transition: all 150ms;
+  transition: border-color 150ms ease-out;
   user-select: none;
   width: ${({ theme }) => theme.spacing.large};
 
+  &:hover {
+    border-color: ${props => (props.checked ? props.theme.colors.primary50 : props.theme.colors.secondary40)};
+  }
```

Comment:
Same issue with checkboxes, doesn't work well with `disabled`.

Reply:
at the moment there doesn't seem to be support for disabled radio buttons (no styling)